airflow/api/common/experimental/trigger_dag.py

```diff
@@ -122,7 +122,12 @@ def trigger_dag(
     dag_model = DagModel.get_current(dag_id)
     if dag_model is None:
         raise DagNotFound("Dag id {} not found in DagModel".format(dag_id))
-    dagbag = DagBag(dag_folder=dag_model.fileloc)
+    from airflow.configuration import conf as global_conf
```

Comment:
```suggestion
    from airflow.configuration import conf
```

Is what we do everywhere else in Airflow.

Reply:
I understand. But I needed a different name because conf parameters is already passed to this method and it denotes conf for dag run (additional parameters)